### ISSUE TITLE
[BUG] vicreo hotkey link?

### ISSUE THREAD

**Issue by jsnyder886** [bitfocus/companion-module-vicreo-hotkey#36]
**Describe the bug**
After upgrading from build 3700 to build 3721, link to vicreo hotkey 2.1.0 no longer  functional.  It does not show up as an error, but it does not initialize either.

**To Reproduce**
Upgrade from 3700 to 3721

**Expected behavior**
Link to initialize

**Screenshots**

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Chrome
- Companion Version: 2.2.0

**Additional context**
Downgrade back to 3700 fixes the issue.

**Comment by JeffreyDavidsz** [comment]
There was a fix, did it help?

**Comment by jsnyder886** [comment]
No, it was the link between companion and vicreo, the actual instance itself.  I've since upgraded to the latest beta well beyond what I referenced originally and it is happy again.  Sorry I don't have the exact new build number I'm on as I'm currently at work.

Thanks for the follow-up!
